scripts/convert-to-celsius.js

```diff
@@ -3,8 +3,31 @@
 console.log('-- loading: convertToCelsius');
 
 
-function convertToCelsius() { }
+function convertToCelsius(f) { 
+  return (f-32) * 5/9;
+}
+
```

Comment:
`const _2_arg = 14;
  const _2_expect = -10;
  const _2_actual = convertToC(_2_arg);
  console.assert(_2_actual === _2_expect, 'Test 2');`

Can you change this  `const _2_actual = convertToC(_2_arg);`  as  `const _2_actual = convertToCelsius(_2_arg);`

Reply:
My function doesn't work with this test. I did it differently. Please check the code. To make it different from the "confirm-ending" function, I wrote a function to convert the temperature.

Reply:
My function doesn't work with this test. I did it differently. Please check the code. To make it different from the "confirm-ending" function, I wrote a function to convert the temperature.

Reply:
Sorry, i saw it now. Will correct immediately